src/Attributes/Operation.php

```diff
@@ -14,42 +14,22 @@ class Operation
     /** @var array<string> */
     public array $tags;
 
-    public ?string $security;
-
     public ?string $method;
 
     public ?array $servers;
 
     /**
      * @param  string|null  $id
      * @param  array  $tags
-     * @param  \Vyuldashev\LaravelOpenApi\Factories\SecuritySchemeFactory|string|null  $security
      * @param  string|null  $method
      *
      * @throws InvalidArgumentException
      */
-    public function __construct(string $id = null, array $tags = [], string $security = null, string $method = null, array $servers = null)
```

Comment:
Please modify this to avoid introducing a breaking change. I would prefer this to be marked as deprecated but still functional for now. :smile: 

Use the "#[Deprecated]" for the argument.